content/sensu-go/5.12/reference/agent.md

```diff
@@ -1158,6 +1172,87 @@ sensu-agent start --statsd-metrics-port 6125
 # /etc/sensu/agent.yml example
 statsd-metrics-port: 6125{{< /highlight >}}
 
+### Allow list configuration
+
+| exec |      |
+----------------------|------
+description           | The command to allow the Sensu agent to run as a check or a hook.
+required              | true
+type                  | String
+example               | {{< highlight shell >}}"exec": "/usr/local/bin/check_memory.sh"{{< /highlight >}}
+
+| sha512 |      |
+----------------------|------
+description           | The checksum of the check or hook command.
```

Comment:
If I understand sensu/sensu-go#3130 correctly, this should be the checksum of the executable, not the command. cc @nikkixdev for confirmation